src/main/java/mate/academy/service/impl/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.service.impl;
+
+import java.util.Optional;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Inject;
+import mate.academy.model.User;
+import mate.academy.service.AuthenticationService;
+import mate.academy.service.UserService;
+
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private UserService userService;
+
+    @Override
+    public User login(String email, String password) throws AuthenticationException {
+        Optional<User> byEmail = userService.findByEmail(email);
+        if (byEmail.isEmpty() || !byEmail.get().getPassword().equals(password)) {
+            throw new AuthenticationException("Can't find User with email = "
+                + email + " and password = " + password);
```

Comment:
```suggestion
            throw new AuthenticationException("Can't find User with email = "
                + email);
```

lets remove sensitive data from exception message